README.md

```diff
@@ -1,34 +1,148 @@
+- [📝 License](#license)
+
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+
+**[Hello Microverse]** Is a welcome to coding in Microverse
+## 🛠 Built With <a name="built-with"></a>
+    - HTML
+    - CSS
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- #
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>  
+
 To get a local copy up and running, follow these steps.
 
-# Prerequisites
-In order to run this project you need:
-# Setup
-Clone this repository to your desired folder:
-# Install
-Install this project with:
-# Usage
-To run the project, execute the following command:
-# Run tests
-To run tests, run the following command:
-# Deployment
+### Prerequisites
+
+In order to run this project:
+- You need to have a Code Editor Installed
+- Make sure Nodejs and NPM is installed in your system for linters
+
+### Setup
+
+To clone this repository to your desired folder, use your code editor terminal or git bash, then type the command below:
+
+- git clone https://github.com/kiplaacollo/Hello-world.git
+
+
+
+### Install
+
+1. Install WebHint:
+
+npm install --save-dev hint@7.x
+
+2. Install StyleLint
+
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x 
+
+
+### Usage
+
+To run the project:
+
+- open cloned folder in your code editor
+
+### Run tests
+
+To run tests and check for errors:
+- After installing the required linter,
+
+npx stylelint "**/*.{css,scss}"
+
+- To fix CSS or SCSS linters error:
+
+npx stylelint "**/*.{css,scss}" --fix
+
+
+### Deployment
+
 You can deploy this project using:
 
-# Authors
-GitHub: 
-Twitter:
-LinkedIn:
+## 👥 Author: <a name="authors"></a>
+
+
+👤 **Kiplaa Collo**
+
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+Give a ⭐️ if you like this project and kindly follow me using my social media handles
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Exceptional appreciations to my coding partners, members of my morning session team and my standup team members.
+- Exceptional appreciation to all the code reviewers at Microverse.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
 
-# Future Features
-Javascript
-# Acknowledgements
-I would like to thank Microverse for giving me this opportunity to learn software development
-# License
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Please always add this section in all your projects in Microverse as it is compulsory. Kindly add it to your README.
> Describe 1 - 3 features you will add to the project in the future. Again this project is simple and nothing is planned to be added in the future, you add anything as placeholders.
```
## 🔭 Future Features <a name="future-features"></a>
- [ ] **[new_feature_1]**
- [ ] **[new_feature_2]**
- [ ] **[new_feature_3]**
```